master/buildbot/steps/source/base.py

```diff
@@ -94,7 +94,25 @@ def __init__(self, workdir=None, mode='update', alwaysUseLatest=False,
         default value will then match all changes.
         """
 
-        LoggingBuildStep.__init__(self, **kwargs)
+        descriptions_for_mode = {
+            "clobber": "checkout",
+            "export": "exporting"}
+        descriptionDones_for_mode = {
+            "clobber": "checkout",
+            "export": "export"}
+
+        if not description:
+            description = [
```

Comment:
Do you think you could fix it?  It does not help with too long lines, but it makes things [a bit] less readable.